linode_api4/groups/object_storage.py

```diff
@@ -299,6 +328,8 @@ def bucket_create(
         label: str,
         acl: ObjectStorageACL = ObjectStorageACL.PRIVATE,
         cors_enabled=False,
+        s3_endpoint: str = None,
+        endpoint_type: ObjectStorageEndpointType = None,
```

Comment:
Could these two be given `Optional[...]` types to be consistent with their default values?